### PR TITLE
Feature/ae 289 retake bug

### PR DESCRIPTION
**Pull request by edugonzlz**
Fixes bugs described in https://z1digitalstudio.atlassian.net/browse/AE-290
- The selfie should happen only once. 
- In principle should be attached to the KYC documents.  Usually they are the first array. Ex: license.
- When the evaluation run, the payload should be document_token_front (front of the license), document_token_back and document_token_selfie.

https://user-images.githubusercontent.com/6910333/181491697-5629cf6a-2944-4d54-85da-666a6705524c.mov

### CODE REVIEW

File: Alloy Example/Alloy Example/ViewController.swift [hunk 1]
```diff
@@ -13,16 +13,15 @@ class ViewController: UIViewController {
...
         
         AlloySettings.configure.apiKey = "028d85e0-aa24-4ca1-99f2-90e3ee3f4e6b"
         AlloySettings.configure.production = true
+        AlloySettings.configure.realProduction = false
         AlloySettings.configure.evaluateOnUpload = false
         
         AlloySettings.configure.steps = [
             .init(orDocumentTypes: [.license, .passport]),
-            .init(orDocumentTypes: [.paystub, .bankStatement, .docW2]),
-            .selfie
```
Comment (mrestuccia) [inline Alloy Example/Alloy Example/ViewController.swift:25]: I see you removed selfie here. Can you confirm selfie is still a parameter that can be on or off?

**Review by mrestuccia — APPROVED** [review]
This looks good to me. 👍 

My only concern is about selfie been a document. Can you confirm that is license and passport are the document options and you add selfie. You ended up sending an evaluation with 
document_token_front
document_token_back
document_token_selfie?

File: Alloy Example/Alloy Example/ViewController.swift (hunk 1 above)
Comment (edugonzlz) [reply Alloy Example/Alloy Example/ViewController.swift:25]: As talked with Antonio.
- Selfie is not configurable by the SDK client.
- Selfie is added as a step by the SDK if document is KYC type
Then, client now can't add Selfie as step. SDK decides.

Comment (edugonzlz) [reply Alloy Example/Alloy Example/ViewController.swift:25]: Selfie is not configured as a parameter. Selfie appears after a KYC document.

Comment (mrestuccia) [reply Alloy Example/Alloy Example/ViewController.swift:25]: OK. Needs to be optional. 
In other words, you need to be able to indicate no selfie.

Comment (mrestuccia) [reply Alloy Example/Alloy Example/ViewController.swift:25]: Of course,  Feel free to separate the issues.

**Comment by edugonzlz** [comment]
Changes about make selfie feature optional by configuration added. 
https://z1digitalstudio.atlassian.net/browse/AE-293

**Review by mrestuccia — APPROVED** [review]

**Pull request merged**